lib/modules/platform/bitbucket/index.ts

```diff
@@ -223,6 +224,17 @@ export async function initRepo({
 
     config.defaultBranch = mainBranch;
 
+    if (bbMendAppDashboardStatus) {
+      logger.debug('Creating branch status for Mend App Repository Dashboard');
+      await setBranchStatus({
```

Comment:
This will set the status every single run. Will it result in duplicates or will the duplicates be ignored server-side by Bitbucket?

And because we also call getStatus() after, it means 2 additional API calls per run even if nothing has changed.

At the very minimum I would like to have repository cache used here to store the last SHA which was set so that we can skip if it's unchanged.

Reply:
It is handled server side, checks are updated in place based on context key. 

I think I'll pivot from this approach that has a bunch of unneeded getStatus and getSha calls and instead simply setStatus with sha=HEAD.

Reply:
Updated to reduce unnecessary API calls since we're always setting HEAD branch status